### ISSUE TITLE
sbt-scoverage fails on Windows OS

### ISSUE THREAD

**Issue by seratch** [scoverage/sbt-scoverage#90]
Although I'm not a Windows OS user, my colleague who works on Windows OS found sbt-scoverage doesn't work on Windows. I'm afraid that file names such as `<empty>` seems not to be allowed on Windows.

- Windows 7
- sbt 0.13.7
- sbt-scoverage 0.99.11 (my colleagues said 1.x has same issue)

```
[info] [scoverage] Waiting for measurement data to sync...
[info] [scoverage] Reading scoverage instrumentation [C:\Users\someone\Desktop\sample\target\dev\scala-2.11\scoverage-data\scoverage.coverage.xml]
[info] [scoverage] Reading scoverage measurements...
[info] [scoverage] Generating Cobertura report [C:\Users\someone\Desktop\sample\target\dev\scala-2.11\coverage-report/cobertura.xml]
[info] [scoverage] Generating XML report [C:\Users\someone\Desktop\sample\target\dev\scala-2.11\scoverage-report/scoverage.xml]
[info] [scoverage] Generating HTML report [C:\Users\someone\Desktop\sample\target\dev\scala-2.11\scoverage-report/index.html]
java.io.IOException: Directory 'C:\Users\someone\Desktop\sample\target\dev\scala-2.11\scoverage-report\<empty>' could not be created
        at org.apache.commons.io.FileUtils.openOutputStream(FileUtils.java:363)
        at org.apache.commons.io.FileUtils.writeStringToFile(FileUtils.java:1928)
        at org.apache.commons.io.FileUtils.write(FileUtils.java:2045)
        at org.apache.commons.io.FileUtils.write(FileUtils.java:1988)
        at scoverage.report.ScoverageHtmlWriter.write(ScoverageHtmlWriter.scala:28)
        at scoverage.report.ScoverageHtmlWriter$$anonfun$write$1.apply(ScoverageHtmlWriter.scala:22)
        at scoverage.report.ScoverageHtmlWriter$$anonfun$write$1.apply(ScoverageHtmlWriter.scala:22)
        at scala.collection.immutable.List.foreach(List.scala:318)
        at scoverage.report.ScoverageHtmlWriter.write(ScoverageHtmlWriter.scala:22)
        at scoverage.ScoverageSbtPlugin$$anonfun$testsCleanup$1$$anonfun$apply$1.apply$mcV$sp(ScoverageSbtPlugin.scala:193)
        at sbt.Tests$$anonfun$Cleanup$1.apply(Tests.scala:62)
        at sbt.Tests$$anonfun$Cleanup$1.apply(Tests.scala:62)
        at sbt.Tests$$anonfun$sbt$Tests$$partApp$1$1$$anonfun$apply$1.apply$mcV$sp(Tests.scala:174)
        at sbt.Tests$$anonfun$sbt$Tests$$fj$1$1.apply(Tests.scala:173)
        at sbt.Tests$$anonfun$sbt$Tests$$fj$1$1.apply(Tests.scala:173)
        at sbt.std.TaskExtra$$anon$1$$anonfun$fork$1$$anonfun$apply$1.apply(TaskExtra.scala:89)
        at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44)
        at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44)
        at sbt.std.Transform$$anon$4.work(System.scala:63)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
        at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
        at sbt.Execute.work(Execute.scala:235)
        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
        at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
        at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)
[error] (dev/scoverage-test:executeTests) java.io.IOException: Directory 'C:\Users\someone\Desktop\sample\target\dev\scala-2.11\scoverage-report\<empty>' could not be created
[error] Total time: 73 s, completed 2015/02/13 10:08:34
```

**Comment by gslowikowski** [comment]
Try with latest 1.0.4 version. This was fixed in 1.0.0 (https://github.com/scoverage/scalac-scoverage-plugin/commit/8a445e36adf52629b6eab46cd3fed6ae0d98b83d)

**Comment by seratch** [comment]
Thank you for your response. 

>my colleagues said 1.x has same issue
I've confirmed her again and above was not a fact.
